File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L6/09/Level6.py
import sys

def rekursif(bil):
    if bil != 0:
        sisa_mod = bil % 10
        print(sisa_mod, end='')
        bil = bil // 10
        rekursif(bil)
    return bil

if __name__ == "__main__":
    print("Enter an integer: ", end='')
    bil = int(input())
    rekursif(bil)
    print()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L6/09/Level6.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function rekursif(bil: number): number {
  if (bil !== 0) {
    const sisaMod = bil % 10;
    console.log(sisaMod);
    return rekursif(Math.floor(bil / 10));
  }
  return bil;
}

rl.question('Enter an integer: ', (input) => {
  const bil = parseInt(input, 10);
  rekursif(bil);
  console.log('');
  rl.close();
});

